components/loading/LoadingBar.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <v-app ref="loaderContainer" class="v-app">
+    <v-progress-linear
+      :active="loading"
+      :indeterminate="indeterminate"
+      :height="4"
+      :bottom="true"
+      :rounded="true"
+      :color="barColor"
+      :value="barValue"
+    ></v-progress-linear>
+  </v-app>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+
+let timeout: number;
+export default Vue.extend({
+  data() {
+    return {
+      loading: false,
+      barColor: 'primary',
+      indeterminate: true,
```

Comment:
Regarding this, I looked a bit into the behavior of the normal Nuxt bar and it seems better to me.

For reference:
> Unfortunately, it is not possible for the Loading component to know in advance how long loading a new page will take. Therefore, it is not possible to accurately animate the progress bar to 100% of the loading time.
> Nuxt's loading component partially solves this by letting you set the duration, this should be set to an estimate of how long the loading process will take. Unless you use a custom loading component, the progress bar will always move from 0% to 100% in duration time (regardless of actual progression). When the loading takes longer than duration time, the progress bar will stay at 100% until the loading finishes.

So instead of being indeterminate, we would go from 0 to 100 in the time set in the config, then stay at 100% until loaded.

For this to work, we'll need to disable the nuxt-axios integration into the loading components, by setting `progress` to `false` in the `axios` key of `nuxt.config.ts` (Since that's what is taking over the loading bar).

Reply:
@MrTimscampi No changes, as soon as I remove all the ``this.$nuxt.$loading.start();`` functions, progress bar doesn't appear.

The [``increase``](https://nuxtjs.org/docs/2.x/configuration-glossary/configuration-loading#using-a-custom-loading-component) method doesn't seem to work as well